com.unity.render-pipelines.high-definition/Documentation~/SGNode-Raytracing-Quality.md

```diff
@@ -0,0 +1,17 @@
+# Raytracing Quality Node
+
+The Raytracing Quality Node allows you to provide a fast implementation of your Shader Graph to be use in GPU heavy raytraced effect to tradeoff accuracy for speed: [Ray-Traced Reflections](Ray-Traced-Reflections.md), [Ray-Traced-Global-Illumination](Ray-Traced-Global-Illumination.md) and [Ray-Traced-Subsurface-Scattering](Ray-Traced-Subsurface-Scattering.md).
+
+## Setup
+
+The Raytracing Quality Node is a builtin keyword and thus need to be created in the Dashboard **Keyword -> Raytracing Quality** of the shader graph then drag and dropped in the working area.
```

Comment:
The Raytracing Quality Node represents a built-in keyword which means, to create one:
1. In the [Blackboard](https://docs.unity3d.com/Packages/com.unity.shadergraph@latest/index.html?subfolder=/manual/Blackboard.html), click the plus (\**+**) button.
2. Select **Keyword > Raytracing Quality**. This creates the keyword and makes it visible on the Blackboard.
3. Now, to use the keyword in your graph, drag it from the Blackboard into the workspace..